docs/data-storage.md

```diff
@@ -41,37 +41,37 @@ gmi.getAllSettings().subtitles;
 Getting a local store instance:
 
 ```javascript
-const localStore = gmi.storage.localKeyValueStore.getStore();
+const store = gmi.storage.localStore.getStore();
```

Comment:
Sorry one last comment: would this be better as `gmi.storage.localStore.get()` ? or does that have conflicts?

Reply:
Going to leave this one as is.